experimental/generation/generator/packages/library/src/processSchemas.ts

```diff
@@ -11,13 +11,37 @@ let allof: any = require('json-schema-merge-allof')
 let clone = require('clone')
 let parser: any = require('json-schema-ref-parser')
 
-type idToSchema = {[id: string]: any}
+/** Mapping from schema name to schema definition. */
+export type idToSchema = {[id: string]: any}
+
+/** 
+ * Find all schemas in template directories.
+ * @param templateDirs User supplied ordered list of template directories.
+ * @returns Object mapping from schema name to schema definition.
+ */
+export async function schemas(templateDirs?: string[]): Promise<idToSchema> {
+    templateDirs = templateDirs || []
+    let templates = await fg.templateDirectories(templateDirs)
+    let schemas = await templateSchemas(templates, feedbackException)
+    return schemas
+}
+
+/** Check to see if schema is global. */
+export function isGlobalSchema(schema: any): boolean {
+    return schema.$global
```

Comment:
$global means ?

Reply:
That the schema is not a per-property schema it is like standard/professional.